js/packages/teams-ai/src/planners/AssistantsPlanner.ts

```diff
@@ -82,17 +102,8 @@ export class AssistantsPlanner<TState extends TurnState = TurnState> implements
             },
             options
         );
-        this._openai = new OpenAI({
-            apiKey: this._options.apiKey
-        });
-    }
 
-    /**
-     * Gets the OpenAI SDK instance being used.
-     * @returns {OpenAI} The OpenAI instance
-     */
-    public get openai(): OpenAI {
```

Comment:
Why is this being deleted? is it somewhere else in the code?

Reply:
since we're not used the library we can't expose the openai client.